torch/_utils.py

```diff
@@ -304,10 +308,6 @@ def _rebuild_sparse_tensor(layout, data):
     raise NotImplementedError(f"rebuilding sparse tensor for layout {layout}")
 
 
-def _rebuild_nested_tensor(buffer, sizes, strides, storage_offsets):
```

Comment:
Were these lines _deleted_? I don't any corrected lines; it appears that these are gone.